data/recipes.yaml

```diff
@@ -44,6 +44,18 @@
   out:
   - [1, wooden gear]
 
+- in:
+  - [1, board]
+  - [2, wooden gear]
+  out:
+  - [1, seesaw]
```

Comment:
Wouldn't it be better to use iron for a seesaw? It feels strange that we won't use it for cutting boards and stuff. :confused: 

I would not mind having a scale just from rocks and a branch, but some joke entity would be fun. :thinking:

Reply:
A seesaw is not for cutting: https://en.wikipedia.org/wiki/Seesaw  It is supposed to be a joke... :smile:   Maybe `seesaw` is a US-specific term?  We could call it a `teeter-totter` instead, maybe that sounds funnier anyway.

Reply:
Also, in terms of gameplay, I think a `comparator` should not be too hard to make (in particular I don't think it should require any iron).  They come in very handy.  You need a comparator if you want to do basic things like recurse on an `int` (so you can tell when it reaches 0).  I also do things like set up a robot to accept harvested trees, and when it notices that the number of trees is over a certain amount, then it crafts them into various products and sends them along.

Reply:
`teeter-totter` sounds better, I also thought seesaw was something for cutting stuff :)

Reply:
OK, done!